### PR TITLE
a formalization of axiomatic reals based on Morse-Kelley set theory

### PR DESCRIPTION
**Pull request by 1DGW**

### CODE REVIEW

**Comment by 1DGW** [comment]
Hello! @palmskog Could you please consider merging this pull request?

File: released/packages/coq-mk-reals-axioms/coq-mk-reals-axioms.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+bug-reports: "https://github.com/1DGW/coq-mk-reals-axioms/issues"
+maintainer: "dgw@bupt.edu.cn"
+authors: [
+  "Wensheng Yu"
+  "Dakai Guo"
+  "Si Chen"
+  "Guowei Dou"
+  "Shukun Len"
+]
+license: "LGPL-2.1"
```
Comment (palmskog) [inline released/packages/coq-mk-reals-axioms/coq-mk-reals-axioms.1.0.0/opam:22]:
```suggestion
license: "LGPL-2.1-only"
```
Based on the license disclaimers.

File: released/packages/coq-mk-reals-axioms/coq-mk-reals-axioms.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  src: "https://github.com/1DGW/coq-mk-reals-axioms/archive/v1.0.0.tar.gz"
+  checksum: "sha512=1ac72d176ed3f52f8ff3eca2adcbfde5b5795af506188dc031870c3f382e35ae665368ac6235d4493113154cdec8d90f25636d9d6b01ec8feaa14099295c1a70"
+}
+
+tags: [
+  "keyword:set theory"
+  "keyword:Morse-Kelley"
+  "keyword:MK"
+  "reals"
+  "category:Mathematics/Logic"
```
Comment (palmskog) [inline released/packages/coq-mk-reals-axioms/coq-mk-reals-axioms.1.0.0/opam:45]:
```suggestion
  "keyword:real numbers"
  "category:Mathematics/Logic/Set theory"
```

**Comment by palmskog** [comment]
@1DGW could you please apply my suggestions in the GitHub review? I will merge after those are applied.

**Comment by 1DGW** [comment]
thank you! @palmskog

**Pull request merged**